cocos/core/data/object.ts

```diff
@@ -244,17 +240,6 @@ class CCObject implements EditorExtendableObject {
         return this._objFlags & CCObject.Flags.AllHideMasks;
     }
 
-    public set replicated (value: boolean) {
-        if (value) {
-            this._objFlags |= IsReplicated;
-        } else {
-            this._objFlags &= ~IsReplicated;
-        }
-    }
-    public get replicated () {
-        return !!(this._objFlags & IsReplicated);
-    }
```

Comment:
Will developer use `replicated` setter or getter in their projects? If it's definitely not, I will approve this PR.

Reply:
They won't, it doesn't have any related implementation